frontend/src/interface/projectInterface.js

```diff
@@ -44,39 +59,63 @@ export function getModels() {
  * Calculates the percentage values and returns them.
  * @param compositionData filtered data from backendconnnect
  */
-export function setMaterialCompositionData(compositionData) {
-    console.log('compositionData');
-    materialDataInPercent = [];
-    materialCompositionData = Array.from(compositionData);
-    console.log(materialCompositionData);
-    for (let i = 0; i < materialCompositionData.length; i++) {
-        materialDataInPercent[i] = Number(materialCompositionData[i]);
+export function setMaterialCompositionData(scenarioName, compositionData) {
+    if (scenarioName === scenarioNames.baseline) {
+        console.log('compositionData');
+        materialDataInPercent = [];
+        materialCompositionData = Array.from(compositionData);
+        console.log(materialCompositionData);
+        for (let i = 0; i < materialCompositionData.length; i++) {
+            materialDataInPercent[i] = Number(materialCompositionData[i]);
+        }
+        console.log(materialDataInPercent);
+    } else if (scenarioName === scenarioNames.modified) {
+        console.log('compositionData');
+        modifiedMaterialDataInPercent = [];
+        modifiedMaterialCompositionData = Array.from(compositionData);
+        console.log(modifiedMaterialCompositionData);
+        for (let i = 0; i < modifiedMaterialCompositionData.length; i++) {
+            modifiedMaterialDataInPercent[i] = Number(modifiedMaterialCompositionData[i]);
+        }
+        console.log(modifiedMaterialDataInPercent);
     }
```

Comment:
When storing the data, we should take this into account 😆 So when scheduling the calculation for the modified scenarioComponent we will have to handle also the storing a little bit different I guess.